scripts/package_building/build_artifacts.sh

```diff
@@ -0,0 +1,404 @@
+#!/bin/bash
+
+set -e
+
+. utils.sh
+
+function install_deps_rhel {
+    #
+    # Installing packages required for the build process.
+    #
+    rpm_packages=(rpm-build rpmdevtools yum-utils ncurses-devel hmaccalc zlib-devel \ 
+    binutils-devel elfutils-libelf-devel openssl-devel wget git ccache bc)
+    yum -y update  
```

Comment:
don't do a yum update, because yum update is the equivalent of apt upgrade and we don t need that. plus, you can nuke all the environment or you need a reboot afterwards.